### PR TITLE
Security definition generation from AuthenticationProviderKey

### PR DESCRIPTION
**Pull request by bodnarkevin**
As suggested in #204 , a simple implemenation of security definition generation from AuthenticationProviderKey

### CODE REVIEW

**Comment by Burgyn** [comment]
I am after appendix surgery.  I'll look into it when I get back from the hospital.

**Comment by bodnarkevin** [comment]
That's fine, get well soon.

File: src/MMLib.SwaggerForOcelot/Transformation/SwaggerJsonTransformer.cs [hunk 1]
```diff
@@ -177,6 +186,34 @@ private bool RemoveMethods(JProperty path, RouteOptions route)
             return path.First.Any();
         }
 
+        private void AddSecurityDefinitions(JProperty path, RouteOptions route)
+        {
+            var authProviderKey = route.AuthenticationOptions?.AuthenticationProviderKey;
+
+            if (string.IsNullOrEmpty(authProviderKey))
+                return;
```
Comment (Burgyn) [inline src/MMLib.SwaggerForOcelot/Transformation/SwaggerJsonTransformer.cs:194]:
```suggestion
           {
                return;
           }
```

**Review by Burgyn — CHANGES_REQUESTED** [review]
Hi,
thank you for this PR.

please follow codefactor bot recommendations. (missing xml comment). After their fixing, I will approve this PR

File: src/MMLib.SwaggerForOcelot/Transformation/SwaggerJsonTransformer.cs (hunk 1 above)
Comment (bodnarkevin) [reply src/MMLib.SwaggerForOcelot/Transformation/SwaggerJsonTransformer.cs:195]: Done

**Comment by bodnarkevin** [comment]
Added comments. Thank You

**Pull request merged**